code/obj/item/building_materials.dm

```diff
@@ -419,7 +419,7 @@ MATERIAL
 					var/turf/T = get_turf(usr)
 					var/area/A = get_area (usr)
 
-					if (!istype(T, /turf/simulated/floor))
+					if (!(istype(T, /turf/simulated/floor) || istype(T, /turf/simulated/space_phoenix_ice_tunnel)))
```

Comment:
Why is this not a normal floor anyways

Reply:
See above comment to Glowbold: "No reason in particular, just happened to go with it. Think it would maybe be better to prevent any possible interaction with floor code elsewhere"